src/style.css

```diff
@@ -0,0 +1,93 @@
+html {
+  font-weight: 300;
+}
+
+body {
+  margin-top: 0;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+.searchBar {
+  width: 100%;
+  background: #fff;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border: 1px solid #e1e7ed;
+  font-family: "Avenir", sans-serif;
+}
+
+.inputWrapper {
+  position: relative;
+}
+
+.searchBar::placeholder {
+  color: #3d4e61;
+}
+
+.form {
+  margin-top: 20px;
+}
+
+.bigInput {
+  height: 70px;
+  font-size: 16px;
+  padding-left: 62px;
+}
+
+.smallInput {
+  height: 42px;
+  font-size: 14px;
+  font-weight: 400;
+  padding-left: 33px;
+}
+
+.icon {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+  width: 19px;
+  height: 19px;
+  background-image: url(images/Search.svg);
+  background-size: cover;
+  background-position: center;
+}
+
+.smallIcon {
+  top: 15px;
+  left: 13px;
+  width: 11px;
+  height: 11px;
+}
+
+.searchBar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.bigInput:focus {
+  font-family: "Avenir", sans-serif;
+  box-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+  text-shadow: 1px 3px 3px #b0b4b9;
+  border-radius: 3px;
+  color: #3d4e61;
+}
+
+.smallInput:focus {
+  font-family: "Avenir", sans-serif;
+  font-style: normal;
+  font-size: 14px;
+  line-height: 19px;
+  color: #3d4e61;
+}
```

Comment:
check font-weight on focus

Reply:
I cant really see the reason why its wrong, I set font-weight to 300. 
Hope its the right solution.